docs/pages/_app.js

```diff
@@ -250,6 +252,8 @@ function AppWrapper(props) {
         {fonts.map((font) => (
           <link rel="stylesheet" href={font} key={font} />
         ))}
+        <meta name="mui:productId" content={productId} />
```

Comment:
Replicate the changes made in https://github.com/mui/material-ui/pull/37593
Without these, all pages would fail, because the code is not null-safe.

Reply:
Awesome, thanks for this 👍

Reply:
Oh, actually, there needs to be more change. It sucks that the logic is duplicated in `_app.js`, we would really need to abstract it https://github.com/mui/mui-x/pull/9451.